src/xunit.performance.execution/BenchmarkTestInvoker.cs

```diff
@@ -190,7 +190,7 @@ protected override void StartMeasurement(int iterationNumber)
 
                     RandomizeMeasurementStartTime();
 
-                    BenchmarkEventSource.Log.BenchmarkIterationStart(BenchmarkConfiguration.Instance.RunId, _testName, iterationNumber);
+                    BenchmarkEventSource.Log.BenchmarkIterationStart(BenchmarkConfiguration.Instance.RunId, _testName, iterationNumber, 0);
```

Comment:
With my suggestion further up in the review to calculate the actual allocated bytes before logging, you should not need this change because the iteration/stop event is the only one that will change.